tests/slow/src/test/scala/tests/feature/SyntaxErrorLspSuite.scala

```diff
@@ -1,7 +1,9 @@
-package tests
+package tests.feature
```

Comment:
Moved it since it depends on publishing the 2.13 version, which slow test already do. Not sure why it worked before.